### ISSUE TITLE
"title" and "text" is mixed up in wcag21-nl.json

### ISSUE THREAD

**Issue by julezrulez** [AccessibilityNL/WCAG21-NL#2]
In guidelines/wcag21-nl.json there is a "title" used in the file where a "text" should be used. See for instance line 8.
I think it happens after every line that starts with "handle".

A good version might be found here (correct me if I'm wrong).
https://github.com/w3c/wcag-em-report-tool/blob/master/app/wcag2spec/wcag2-nl.json
Edit: Nope that one isn't correct either.

**Comment by rvantonisse** [comment]
Hey @julezrulez ,

Better late than sorry! 😉 

The json is generated by this XML / Ant project command at w3c/wcag: https://github.com/w3c/wcag/blob/63f597f6b9fc52529970bdbdb24e9f75594bd350/build.xml#L348

Hope that helps answering the why at least.
